src/connect.ts

```diff
@@ -0,0 +1,103 @@
+import { Coin, DirectSecp256k1HdWallet } from "@cosmjs/proto-signing";
+import { Block, calculateFee, GasPrice } from "@cosmjs/stargate";
+import {SigningStargateClient, StargateClient } from "@cosmjs/stargate";
+import { ArchwayClient, ArchwayTxFilter, ArchwayTXSearch } from "./query";
+import { SigningCosmWasmClient, CosmWasmClient } from "@cosmjs/cosmwasm-stargate";
+import {uploadContract} from "./wasm"
+// Wrapper for connecting to RPC client from COSM JS
+// Can either connect to testnet or mainnet
+// TODO: add mainnet
```

Comment:
if you add `TODO` please create an issue and add the issue number in the comment